src/components/BookCard.tsx

```diff
@@ -0,0 +1,17 @@
+import { IBook } from '../interfaces/books';
+
+const BookCard = (key: any, book: IBook) => {
+
+    return (
+        <div key={key} className="book">
```

Comment:
Nit - `key` is not a real prop that you should read and it is unnecessary in this context, it should only be used when returning an array of elements. This component returns a single element